katello_certs_tools/fileutils.py

```diff
@@ -79,17 +79,15 @@ def rotateFile(filepath, depth=5, suffix='.', verbosity=0):
     """
 
     # check argument sanity (should really be down outside of this function)
-    if not filepath or type(filepath) != type(''):
+    if not filepath or isinstance(filepath, str):
```

Comment:
```suggestion
    if not filepath or not isinstance(filepath, str):
```

otherwise you reverse the logic of the original code